letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -15,6 +15,11 @@ set -e  # Work even if somebody does "sh thisscript.sh".
 
 # Note: you can set XDG_DATA_HOME or VENV_PATH before running this script,
 # if you want to change where the virtual environment will be installed
+
+# HOME might not be defined when being run through something like systemd
+if [ -z "$HOME" ]; then
+  HOME=/root
```

Comment:
Good point. Do you know if `~root` is POSIX compliant? `certbot-auto` gets run in all kinds of shells; bash, dash, busybox, csh, tcsh, etc.